test/hotspot/jtreg/vmTestbase/nsk/share/jdi/EventFilters.java

```diff
@@ -337,6 +337,9 @@ public static boolean filtered(Event event) {
         if (event.toString().contains("JFR request timer"))
             return true;
 
+        if (event.toString().contains("ForkJoinPool"))
+            return true;
+
```

Comment:
As I understand, this is to filter out events for carrier threads.
Is it right? If so, it'd be good to add a short comment to make it clear.

Reply:
Ok. It's fixed now.